ovirtlago/testlib.py

```diff
@@ -209,7 +209,7 @@ def _instance_of_any(obj, cls_list):
     return any(True for cls in cls_list if isinstance(obj, cls))
 
 
-def assert_equals_within(func, value, timeout, allowed_exceptions=None):
+def assert_equals_within(func, value, timeout, allowed_exceptions=None, initial_wait=10):
```

Comment:
10 seconds can be quite a lot for initial waiting, no?
Any failure -> 10 seconds penalty to start with.